packages/redux/src/contents/serverInitialState.ts

```diff
@@ -24,12 +24,14 @@ const serverInitialState: ServerInitialState = ({ model }) => {
   const { searchContentRequests, slug, seoMetadata, subfolder } = model;
 
   const contents = searchContentRequests.reduce((acc, item) => {
-    const {
-      filters: { codes, contentTypeCode },
-      searchResponse,
-    } = item;
+    const { searchResponse } = item;
 
-    const hash = generateContentHash({ codes, contentTypeCode });
+    if (!searchResponse.entries[0]) return acc;
```

Comment:
Prefer this form please:

```suggestion
    if (!searchResponse.entries[0]) { 
        return acc;
     }
```

Reply:
Resolved ✅